app/App/Main/Account/SignerStatus/index.js

```diff
@@ -76,7 +76,9 @@ class SignerStatus extends React.Component {
           this.props.hideSignerStatus(true)
         }
       })
-      this.inputRef.current.focus()
+      if (this.inputRef.current) {
```

Comment:
The component will sometimes render as null. This isn't the React way to go about things but rather than rework the parent component I decided to reduce the scope and guard it